docker-compose.yml

```diff
@@ -216,10 +216,10 @@ services:
       ORB_DRIVER_HOST_URL: ${ORB_DRIVER_HOST_URL}
       ORB_DRIVER_TLS_SYSTEMCERTPOOL: ${ORB_DRIVER_TLS_SYSTEMCERTPOOL}
     ports:
-      - "8121:8121"
+      - "8122:8121"
```

Comment:
shouldn't this be `8122:8080`?

Reply:
The container port (2nd) can be freely chosen by the implementers and did:orb is listening on [port 8121](https://github.com/decentralized-identity/universal-resolver/blob/5d9655bf44ec23afb667cf2f24bac7b71bcf1372/config.json#L248)

Reply:
I see, thanks for the clarification